test/api-helper/misc/job/createJob.test.ts

```diff
@@ -143,4 +143,38 @@ describe("createJob", () => {
         assert.deepStrictEqual(result, { id: "123456" });
     });
 
+    it("should not reject invalid parameters", async () => {
```

Comment:
Perhaps "should allow an empty parameter array"

Reply:
I removed auto-merge in case you want to change that.

Reply:
Good point!